src/token_generator.js

```diff
@@ -36,17 +36,17 @@ function tokenGenerator(identity = 0) {
   if (config.TWILIO_CHAT_SERVICE_SID) {
     // Create a "grant" which enables a client to use IPM as a given user,
     // on a given device
-    const ipmGrant = new IpMessagingGrant({
+    const chatGrant = new ChatGrant({
       serviceSid: config.TWILIO_CHAT_SERVICE_SID
     });
-    token.addGrant(ipmGrant);
+    token.addGrant(chatGrant);
   }
 
   if (config.TWILIO_SYNC_SERVICE_SID) {
-    // Create a "grant" which enables a client to use Sync as a given user,
-    // on a given device
+    // Point to a particular Sync service, or use the account default to
+    // interact directly with Functions.
```

Comment:
Should this actually be Sync here (instead of Functions?)

Reply:
I wonder where this text came from, actually; I don't remember writing it. :D 

This is a weird place to mention functions. Will fix…